ws-nextjs-app/cypress/e2e/languagesPage/index.cy.ts

```diff
@@ -0,0 +1,45 @@
+import { HOME_PAGE, LIVE_PAGE } from '#app/routes/utils/pageTypes';
+import * as assertions from './assertions/languagesPage';
```

Comment:
Let's import the assertions we need by name
```suggestion
import { assertPidginHomepage, assertWSLanguagesPage } from './assertions';
```